src/Sylius/Bundle/AdminBundle/package.json

```
@@ -14,6 +14,7 @@
     "@babel/preset-env": "^7.25.8",
     "@hotwired/stimulus": "^3.0.0",
     "@popperjs/core": "^2.11.8",
+    "@sylius/admin-bundle": "file:Resources/assets/",
```

1. Should we keep the namespace the same at for `@sylius-ui/admin`?
2. Is second package necessary?
3. If second `package.json` is necessary, then should it published or marked as `private: true`?

```suggestion
    "@sylius-ui/admin-bundle": "file:Resources/assets/",
```

---

I created a new package.json inside Resources/assets because it's where Flex look for importing controllers https://github.com/symfony/flex/blob/2.x/src/PackageJsonSynchronizer.php#L396

I keep the old one to not break build for existing shop

For the namespace, I thought it was correct to reflecting the actual bundle name, however, referring this doc (https://symfony.com/doc/current/frontend/create_ux_bundle.html) it could be name `@sylius/admin` but i think it doesn't matter